networking/using-ptp.adoc

```diff
@@ -5,13 +5,13 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: Precision Time Protocol (PTP) hardware
+:FeatureName: Precision Time Protocol (PTP) hardware with single NIC and boundary clock
```

Comment:
Can the wording be more clear that it's a boundary clock configured on a NIC?

Reply:
Good point. What about "Precision Time Protocol (PTP) hardware with single NIC configured as boundary clock".